lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -445,7 +445,8 @@ protected function transformSignup($data, $current) {
    */
   protected function transformUser($data) {
     return [
-      'id' => $data->id,
+      'id' => $data->drupal_id,
```

Comment:
What we could do is add a `legacy_id` property and populate the Drupal one there and leave the NS one as just `id` like it was before...
@DFurnes thoughts?